content/ecosystem/sdks/sdks.toml

```diff
@@ -660,3 +660,16 @@ featured_in = []
 description = """
 Matrix client SDK in C#
 """
+
+[[sdks]]
+name = "POSIX Shell Matrix SDK"
+maintainer = "wj25czxj47bu6q"
+maturity = "Beta"
+language = "POSIX Shell"
+licence = "MIT-Modern-Variant"
```

Comment:
This seems to not be the case according to https://gitlab.com/matrix-mellifluousness/sdk/-/blob/ec678e2a9a7011d8031a5654a7743cb7cb6245e5/LICENSE.txt which claims "all rights reserved". Which also to me reads like a non OSI license even though it might be based upon MIT.

Reply:
I got it from
![image](https://github.com/user-attachments/assets/dc5aad52-05ac-489e-820b-c9936e497d56)

Reply:
The "All rights reserved" clashes with the rest of the license. I'd suggest to ask them to update their license so we can add it to m.org